mp/src/game/client/hl2mp/hl2mp_hud_target_id.cpp

```diff
@@ -12,7 +12,7 @@
 
 using namespace vgui;
 
-#ifdef NEO
+#ifndef NEO
 DECLARE_NAMED_HUDELEMENT(CTargetID, TargetID);
 #endif
```

Comment:
Question for reviewers: Should this be removed entirely as `CTargetID` is not referenced anywhere else or just left as it is?

Reply:
I wouldn't mind keeping it in the source in case someone wants to cross reference the original implementation for something, but I guess we could drop it from CMakeLists if it's not used anywhere.